htdocs/modules/profile/sql/mysql.sql

```diff
@@ -27,7 +27,7 @@ CREATE TABLE `profile_field` (
   `step_id`             smallint(3) unsigned    NOT NULL default '0',
   
   PRIMARY KEY  (`field_id`),
-  UNIQUE KEY `field_name` (`field_name`),
+  UNIQUE KEY `field_name` (`field_name`(100)),
```

Comment:
This bothers me, since the key is supposed to be unique, it is only a matter of time until a collision occurs,

But, after deeper review I found that the 'field_name' column is used as a MySQL column name. A column name is limited to 64 characters in MySQL, as are virtually all identifiers in MySQL

The best solution here is to shrink the.'field_name' declaration to varchar(64) and undo the change to the UNIQUE KEY declaration.

Also need to consider new needs in upgrade

Reply:
Done